### PR TITLE
Network data

### PR DESCRIPTION
**Pull request by degleris1**
This PR adds network case files, demand time series case files, and solar time series case files. It also adds code to quickly load these files.

- `data/` now contains various case files. They are quite small, so I think it's reasonable to keep them on git.
- `parse_files.jl` now contains additional code to parse various datafiles.
- Various outdated notebooks have been deleted. Some have been renamed. I think the prefixes `demo_`, `experiment_`, `scratch_`, and `test_` should be used to denote the purpose of the notebook. I have also gradually switched over to Pluto notebooks, which are reactive and save as plain text files.
- I have added a new Pluto notebook `experiment_storage_pentration.jl` that runs the storage penetration experiment on real data.

### CODE REVIEW

File: notebooks/experiment_storage_penetration.jl [hunk 1]
```diff
@@ -0,0 +1,329 @@
...
+# ╔═╡ cfcba5ad-e516-4223-860e-b1f18a6449ba
+begin
+	plt1 = plot(demand_data[:, rand(1:n_demand, 5)], lw=2, palette=:Blues)
+	plt2 = plot(renew_data, lw=2, palette=:Reds)
+	
+	plot(plt1, plt2, layout=(2, 1), size=(600, 300))
+end
+
+# ╔═╡ c82ef027-740a-49b1-93d2-1554c411a896
+renewable_pentration = 0.5
```
Comment (LucasFuentesValenzuela) [inline notebooks/experiment_storage_penetration.jl:95]: fix typo

File: notebooks/experiment_storage_penetration.jl [hunk 2]
```diff
@@ -0,0 +1,329 @@
...
+### Carbon emissions data
+"""
+
+# ╔═╡ 897cc3a0-66cb-417d-81d5-20036b32dd99
+net.gmax
+
+# ╔═╡ e8ee5cbb-4afc-4737-b006-90071f6138cd
+begin
+	Random.seed!(3)
+	emission_rates = 1000 .+ 1000*rand(length(net.gmax))
```
Comment (LucasFuentesValenzuela) [inline notebooks/experiment_storage_penetration.jl:139]: Are these emission rates the same as the carbon costs? Or do they represent something else?

File: notebooks/experiment_storage_penetration.jl [hunk 3]
```diff
@@ -0,0 +1,329 @@
...
+			mef_times=mef_times,
+			emission_rates=emission_rates,
+			d_dyn=d_dyn,
+		)
+		
+		results = zeros(n, length(mef_times), length(storage_penetrations))
+		for (ind_s, s_rel) in enumerate(storage_penetrations)
+
+			# Construct dynamic network
+			C = d_peak * s_rel
```
Comment (LucasFuentesValenzuela) [inline notebooks/experiment_storage_penetration.jl:234]: The definition of storage penetration is therefore a scaling wrt the peak demand? That makes sense i believe. 

In this case, there is no reason to be up-limited by `1`, correct?

File: src/parse_data.jl [hunk 4]
```diff
@@ -125,4 +125,135 @@ function load_case(name, agg_nodes, B, nodes; resources=BASE_RESOURCES)
...
+end
+
+"""
+    load_synthetic_network(case_name)
+"""
+function load_synthetic_network(case_name)
+    # Load network data
+    network_data = parse_file(joinpath("../data", case_name));
+	net = make_basic_network(network_data)
+	make_per_unit!(net)
```
Comment (LucasFuentesValenzuela) [inline src/parse_data.jl:137]: I like this indentation style. In Julia, what does it mean? Is it like a paragraph?

File: src/parse_data.jl [hunk 5]
```diff
@@ -125,4 +125,135 @@ function load_case(name, agg_nodes, B, nodes; resources=BASE_RESOURCES)
     end
     
     return d, g, case
+end
+
+"""
+    load_synthetic_network(case_name)
```
Comment (LucasFuentesValenzuela) [inline src/parse_data.jl:131]: Can you add one line of `help`/docstring? Same comment for functions below

**Review by LucasFuentesValenzuela — COMMENTED** [review]
In general looks good. A few small comments/questions added

File: src/parse_data.jl (hunk 4 above)
Comment (degleris1) [reply src/parse_data.jl:137]: Ah, just a typo :)

**Pull request merged**